### ISSUE TITLE
frame-title should flag frames with negative tabindex as incomplete

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#3484]
[Inapplicable example 3](https://wai-wcag-act-rules.netlify.app/standards-guidelines/act/rules/cae760/proposed/#inapplicable-example-3) should not fail the frame-title rule:

```html
<iframe tabindex="-1" src="/test-assets/SC4-1-2-frame-doc.html"> </iframe>
```

Instead of failing these, I think we should report them as incomplete. It largely depends on what you're doing with this element whether or not it needs an accessible name.

**Comment by straker** [comment]
Work was completed in https://github.com/dequelabs/axe-core/commit/f61f825843ec323fbe7ebb7096bf4c7341f10feb but accidentally pushed to develop directly instead of a pr.

**Comment by padmavemulapati** [comment]
Validated with the latest develop branch code base, and `frame-title` with negative tabindex in going in `inapplicable`


![Image](https://user-images.githubusercontent.com/47409579/187854552-6b535319-bb47-45a6-b1cb-a1e024236fdf.png)
